### PR TITLE
t4k_common new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: dev-games/t4k_common/patches/t4k_common-0.1.1.patchset [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+
+diff --git a/src/linebreak/linebreak.c b/src/linebreak/linebreak.c
+index f9c88a1..9cbf4d2 100644
+--- a/src/linebreak/linebreak.c
++++ b/src/linebreak/linebreak.c
+@@ -22,6 +22,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.  */
+  */
+ 
+ #if defined BUILD_MINGW32 && !defined ICONV_CONST
```
Comment (pulkomandy) [inline dev-games/t4k_common/patches/t4k_common-0.1.1.patchset]: This breaks the mingw build, I think?

File: dev-games/t4k_common/patches/t4k_common-0.1.1.patchset [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
++#elif defined(__HAIKU__) && !defined ICONV_CONST
```
Comment (Begasus) [inline dev-games/t4k_common/patches/t4k_common-0.1.1.patchset]: Yeah, while I was out today, thought about that option too ... will look into it :)

**Comment by Begasus** [comment]
Upped the recipe, still have to check on on x86_64

**Comment by Begasus** [comment]
vmhaiku
Buids and installs on gcc2 hybrid and x86_64 :)

**Pull request merged**